ex03.ex

```diff
@@ -77,7 +80,9 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([ x | _ ], x), do: true
```

Comment:
## -1

There's a bug here. I can write

~~~
list_contains(99, 123)
~~~ 

and it just returns `false`